sympy/matrices/dense.py

```diff
@@ -53,6 +45,9 @@ def __eq__(self, other):
         except AttributeError:
             return False
 
+    def __ne__(self, other):
```

Comment:
If I recall correctly, I had to add that to get it working in python2.  Python3 properly negates `__eq__`.